nrt/data/DynaFlow/LoadRestorativeWithLimits/reference/outputs/curves/curves.csv

```diff
@@ -1,34 +1,34 @@
-time;_Load__10_PPu_value;_Load__10_QPu_value;NETWORK__BUS___10_TN_Upu_value;_Load__10_UfRawPu_value;_Load__10_UfPu_value;
```

Comment:
How does this change affect the nrt of dynawo-rte ?

Reply:
very small changes in values